doc/intro.md

```diff
@@ -113,6 +112,7 @@ Some parameters in the model can be specified by the user. The parameters, with
 * Start & end years of year range to average over when projecting domestic out-migrants; migration out of an area (2003-2014)
 * Start & end years of year range to average over when projecting international in-migrants; migration into an area (2003-2014)
 * Start & end years of year range to average over when projecting international out-migrants; migration out of area (2003-2014)
+* Brexit parameter (1) **N.B.**: Takes a value 0-1. A value of one maintains the current estimated international in-migrant population projection.  
```

Comment:
Can this be modelled as a more general purpose internation in-migration multiplier that we nickname the 'brexit parameter' rather than advertising it as such? I guess in other contexts, it might make sense that we allow for positive international in-migration influence